.github/actions/action-lacework-scan/action.yml

```diff
@@ -0,0 +1,130 @@
+      BUILD_REPORT_FILE_NAME: ${{ steps.prepare-input.outputs.safe-image-id }}.lacework-report.html
+      LW_SCANNER_ENABLE_DEBUGGING: ${{ inputs.debug-mode }}
+
+
+  - name: Process result
+    id: process-result
+    shell: bash
+    run: |
+      fullJson=$(find ./evaluations -name 'evaluation_*.json' 2>/dev/null)
+      echo "Parsing result from: $fullJson"
+
+      # Check image is the same.
+      imageName=$(cat $fullJson | jq -r '.cve.image.image_info.repository')
+      imageTag=$(cat $fullJson | jq -r '.cve.image.image_info.tags[0]')
+      if [ "$imageName" != "${{ steps.prepare-input.outputs.image-name }}" ] || [ "$imageTag" != "${{ steps.prepare-input.outputs.image-tag }}" ]; then
+        >&2 echo "Image mismatch: ${imageName}:${imageTag} != ${{ steps.prepare-input.outputs.image-name }}:${{ steps.prepare-input.outputs.image-tag }}"
+        exit 1
+      fi
+      
+      # Filter JSON by deleting vulnerabilities with unwanted severities.
+      filteredJson="${{ steps.prepare-input.outputs.safe-image-id }}.filtered.json"
+      jq 'del(.cve.image.image_layers[].packages[].vulnerabilities[] | select(.severity != "Critical" and  .severity != "High" ))
+          | del(.cve.image.image_layers[].packages[] | select(.vulnerabilities == []))
+          | del(.cve.image.image_layers[] | select(.packages == []))' \
+        $fullJson > $filteredJson
+      
+      # Convert JSON into a simplified JSON.
+      resultJson="${{ steps.prepare-input.outputs.safe-image-id }}.result.json"
+      jq '{
+            image: "${{ inputs.image }}",
+            stats: {
+              Critical: { total: .cve.critical_vulnerabilities, fixable: .cve.critical_fixable_vulnerabilities },
+              High: { total: .cve.high_vulnerabilities, fixable: .cve.high_fixable_vulnerabilities },
+              Medium: { total: .cve.medium_vulnerabilities, fixable: .cve.medium_fixable_vulnerabilities },
+              Low: { total: .cve.low_vulnerabilities, fixable: .cve.low_fixable_vulnerabilities },
+              Info: { total: .cve.info_vulnerabilities, fixable: .cve.info_fixable_vulnerabilities }
+            },
+            includedSeverities: [ "Critical", "High" ], 
+            cves: [.cve.image.image_layers[].packages[] | . as $p | .vulnerabilities[]
+              | {cveId: .name, severity: .severity, package: $p.name, currentVersion: $p.version, fixVersion: .fix_version, status: .status, link: .link}]
+              | sort_by(.severity, .cveId)
+          }' $filteredJson > $resultJson
+      
+      # Set multiline output.
+      echo 'result-json<<EOF' >> $GITHUB_OUTPUT
+      cat $resultJson >> $GITHUB_OUTPUT
+      echo 'EOF' >> $GITHUB_OUTPUT
+
+  - name: Print result json (debug mode)
+    if: inputs.debug-mode == 'true'
+    shell: bash
+    run: |
+      echo "*** RESULT JSON:"
+      cat ${{ steps.prepare-input.outputs.safe-image-id }}.result.json
+
+  - name: Upload results to artifacts (debug mode)
```

Comment:
Is this still useful? I guess we end up with the artifact files in the workflow summary?

Reply:
Not needed anymore, removed. :+1: